app/enterprise/0.32-x/getting-started/adding-your-api.md

```diff
@@ -93,7 +93,7 @@ Now that you've added your API to Kong EE, let's learn how to enable plugins.
 
 Go to [Enabling Plugins &rsaquo;][enabling-plugins]
 
-[API]: /latest/admin-api
+[API]: /0.13.x/admin-api
 [enabling-plugins]: /enterprise/{{page.kong_version}}/getting-started/enabling-plugins
-[proxy-port]: /latest/configuration/#nginx-section
+[proxy-port]: /0.13.x/configuration/#nginx-section
```

Comment:
This section doesn't appear on the preview.

Reply:
https://docs.konghq.com/0.13.x/configuration/#nginx-section